server/errorHandler.test.ts

```diff
@@ -1,20 +1,14 @@
-import type { Express } from 'express'
 import request from 'supertest'
 import { appWithAllRoutes } from './routes/testutils/appSetup'
-
-let app: Express
-
-beforeEach(() => {
-  app = appWithAllRoutes({})
-})
+import DashboardController from './controllers/dashboardController'
 
 afterEach(() => {
   jest.resetAllMocks()
 })
 
 describe('GET 404', () => {
   it('should render content with stack in dev mode', () => {
-    return request(app)
+    return request(appWithAllRoutes({ controllers: { dashboardController: new DashboardController() } }))
```

Comment:
Isn't the object here and below duplicating code in `server/controllers/index.ts`? Can't we just import from that instead of importing the `DashboardController` and recreating it?